video.py

```diff
@@ -0,0 +1,45 @@
+import requests
+import datetime
+from rental import RentalOps
+
+class VideoOps:
+    def __init__(self, url = "https://retro-video-store-api.herokuapp.com", selected_video=None):
+        self.url = url
+        self.selected_video = selected_video
+
+    def add_video(self, title, release_date, total_inventory):
+        query_params = {
+            "title": title,
+            "release_date": release_date,
```

Comment:
hm... does release_date need to be in a particular form? I got a 404 when I entered a date like 08232021